pages/_app.tsx

```diff
@@ -1,11 +1,15 @@
-import Layout from '@/components/Layout'
+import Header from '@/components/Header'
+// import Layout from '@/components/Layout'
 import '@/styles/globals.css'
 import type { AppProps } from 'next/app'
 
 export default function App({ Component, pageProps }: AppProps) {
+  const categories = ['documentary', 'industry', 'festival', 'culture', 'nature', 'sports', 'broadcast', 'creative']
```

Comment:
What is the `categories` array for?

Reply:
I don't have a use for it yet really. I'll just get rid of it for now